Cartfile

```diff
@@ -1,8 +1,8 @@
 # Used for HTTP routing
-github "marekcirkos/RoutingHTTPServer"
+github "shvul/RoutingHTTPServer"
```

Comment:
I'd rather move this component to appium org

Reply:
Me too. He has a fork of the Cocoa HTTP server too.